### PR TITLE
Finalized QoL storage hotkeys

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request
This PR adds Shift+Q to take from and put into suit storage, and Shift + B to take from and put into your backpack or bag

## Why It's Good For The Game

These will make it faster for you to pull out your deathmurder gun 9000 and blow away ironhammer 

## Changelog
:cl:
add: Added new hotkeys, shift + B for taking from and putting into the backpack, and Shift + Q for taking from and placing into suit storage
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

Any content that lets me validhunt more efficiently is good for the game

But really, having a more efficient UI and hotkey system is always, always good.

Still needs Maintainer approval.

**Review by Hopekz — APPROVED** [review]

File: code/modules/mob/inventory/equip.dm
Comment (TorinoFermic) [inline code/modules/mob/inventory/equip.dm:154]: You can swap both code blocks and leave the bottom part with a simple else

Comment (TorinoFermic) [inline code/modules/mob/inventory/equip.dm:154]: Like this

```suggestion

	if(Item == src.s_store) 
		if(put_in_active_hand(Item))
			return TRUE
	else
		if(equip_to_slot_if_possible(Item, slot_s_store))
			return TRUE
```

File: code/modules/mob/living/carbon/human/inventory.dm
Comment (TorinoFermic) [inline code/modules/mob/living/carbon/human/inventory.dm:45]:
```suggestion
	var/obj/item/I = get_active_hand()
	if(I)
		if(src.s_store)
			to_chat(src, SPAN_NOTICE("You have no room to equip or draw."))
		else
			equip_to_from_suit_storage(I)
	else if ( src.s_store )
		equip_to_from_suit_storage(src.s_store)
	else
		to_chat(src, SPAN_NOTICE("You are not holding anything to equip or draw."))
	return
```

**Pull request merged**
